JS Basics/Objects and Array methods/Problem_7/index.js

```diff
@@ -0,0 +1,34 @@
+function createPerson(firstname, lastname, age, gender) {
+    return {
+        firstname: firstname,
+        lastname: lastname,
+        age: age,
+        gender: gender ? 'female' : 'male'
+    }
+}
+
+function generatePerson() {
+    const people = [];
+    const firstnameFemale = ["Ally", "Lyla", "Kylia", "Daniela", "Reily", "Millie", "Hailey", "Miranda", "Zoe", "Molly"];
+    const firstnameMale = ["Alex", "Aaron", "Austin", "Bryce", "Jacob", "Diego", "Isaac", "Edward", "Sam", "Aiden"];
+    const lastname = ["Abbott", "Abraham", "Adams", "Aldridge", "Allen", "Anderson", "Armstrong", "Banks", "Barnett", "Barrow"];
+
+    for(let i = 1; i <= 10; i++) {           
```

Comment:
No usage of regular for loops.
Think how the data may be presented so that you can benefit from build in array iterator methods such as array.foreach